changelogs/fragments/67037-openssl_certificate-acme-directory.yaml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
```

Comment:
About the changelog filename: the number should be the PR #, i.e. the filename should be `67109-openssl_certificate-...`

Reply:
Thank you. I've updated the PR again. I also updated the commit message to contain a more detailed message as I see others do.

Reply:
The commit message is autogenerated from the PR's title and the commit messages of its individual commits.